lab-abigail/test/greet-test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const assert = require('assert');
+const greeting = require('../lib/greet.js');
+
+
+describe('greet.js', function() {
+  describe('#greet', function() {
+    it('should return, hello Abigail', function() {
+      assert.equal(greeting.greet('Abigail'), 'hello Abigail');
+    });
+    it('should return a string', function() {
+      assert.equal(typeof greeting.greet('Abigail'), 'string');
+    });
+    it('entered name matches', function() {
+      assert.equal(greeting.greet(process.argv[2]), `hello ${process.argv[2]}`);
```

Comment:
Good job trying to test for ``process.argv[2]``, but this isn't quite doing what you think it's doing. It passes because the only thing logged out is ``hello``, so of course they're equal to each other.

I know it was tricky to test for ``process.argv[2]`` in this case. One way to do it is to set ``process.argv[2]`` to something like:

```
process.argv[2] = 'judy';
```

Then Line 16 would be:

```
assert.equal(greeting.greet(process.argv[2]), 'hello judy');
```
See me if this doesn't make sense. :)